packages/react-components/src/Concepts/Icons/Icon.stories.mdx

```diff
@@ -0,0 +1,31 @@
+import { Meta } from '@storybook/addon-docs';
+import { ReactIconGrid } from './ReactIconGrid';
+
+<Meta title="Concepts/Developer/Icons" />
+
+## Icons
+
+The Fluent UI SVG icon set is released under the MIT License.
+
+The `@fluentui/react-icons` package is an icon offering from the [fluentui-system-icons](https://github.com/microsoft/fluentui-system-icons/tree/master/packages/react-icons) design set. It contains SVG-based icons wrapped in React components. Because each icon is its own component, you only need to import and bundle the icons used in your application.
+
+There are two different styles of each icon, `Filled` and `Regular`. These are by default sized to `1em`, and can be scaled up or down to suit your needs.
+
+To use these icons, simply import them as `import { [IconName][Style] } from @fluentui/react-icons`. For example:
+
+```tsx
+import { AccessTimeFilled } from '@fluentui/react-icons';
+```
+
+If you know what size you want to use for your icons, and they don't need to be resized dynamically, it is recommended to use the sized icons. These are the same icons as in the general case, but in multiple different sizes, built to look pixel-perfect at each size.
+You can import the sized icons in a similar way:
+
+```tsx
+import { AccessTime24Filled } from '@fluentui/react-icons';
+```
+
+Each icon also accepts `className` and `primaryFill` props for styling.
+
+### Available Icons
+
+<ReactIconGrid />
```

Comment:
To lazy load this you can remove the import from the top of the file, then do this (React is already available globally in mdx files):
```
<React.Suspense fallback="Loading...">
  {React.createElement(React.lazy(() => import('./ReactIconGrid')))}
</React.Suspense>
```

Reply:
Also in ReactIconGrid.tsx you'll need to add `export default ReactIconGrid;` at the bottom.